content/about/index.md

```diff
@@ -1,9 +1,72 @@
 ---
-title: "About"
+title: "About Us"
 author: "Zap"
 date: 2018-05-25T18:07:23+05:30
-cover_image: "/images/image1.jpeg"
+cover_image: "/images/aboutus/zap.jpg"
 ---
 
-### Coming Soon....
+<div class="container">
+
+	<p>
+		<i>"Every word is not an abbreviation"</i>. Zap is a group of four members. We are studying in Northern India Engineering College and pursuing Btech degree. We all love to implement our theoretical knowledge into our real life. We also love to write blogs on what we learn. We have also represented India at Oulu HealthHack Hackathon at Finland. We are on 100Days of Code Challenge.
```

Comment:
Change _Zap_ to `<strong>Zap!</strong>`

I have made some edition to introduction text.

Every word is not an abbreviation". Zap! is a team of four members passionate about their work, exploring new domains of technology. We are Computer Science students pursuing B.Tech from Northern India Engineering College. We all love to implement our theoretical knowledge into our real life, learning along the way. We love to write blogs on what we learn and experiment. We represented India at Oulu HealthHack in Finland. Currently we are on #100Days of Code Challenge.